_data/simple-icons.json

```
@@ -3409,7 +3409,8 @@
         {
             "title": "Figma",
             "hex": "F24E1E",
-            "source": "https://brand.figma.com/icon.html"
+            "source": "https://brand.figma.com/Figma-Brand-Assets.zip",
```

According to the [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), the link to the branding page and not the link to the ZIP archive should be used:

> If the SVG is sourced from:
>
> - **Branding page**: For an SVG from a branding page the source URL should link to the branding page and not the image, PDF, or archive (such as `.zip`) file.

```suggestion
            "source": "https://www.figma.com/using-the-figma-brand/",
```